go.mod

```diff
@@ -42,29 +42,40 @@ require (
 	golang.org/x/text v0.24.0
 )
 
+require (
```

Comment:
it's nice if you can keep the indirect deps in the same block

Reply:
huh, that's weird. i just regenerated this with `go mod tidy` when rebasing to resolve the conflicts with `HEAD`.

why would it have created this second block for these indirect deps?

Reply:
it seems like the `tool` directive prevents `go mod tidy` from "using" the `require` block below it. that's unexpected

Reply:
manually reordered the directives in f281039